assets/i18n/zh-tw.json

```
@@ -0,0 +1,269 @@
+    "dgc-stores-title": "商店", 
+    "dgc-stores-buy": "用狗狗幣支付",
+    "dgc-footer-credits": "Created by <a href=\"https://ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Website Design by <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo by <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
+    "dgc-footer-copyright": "© The Ðogecoin Project", 
+    "dgc-browser-modal-title": "選擇網路錢包：", 
+    "dgc-browser-modal-online": "網路錢包是使用狗狗幣最快速且簡單的方法，但安全性會不如在您的離線電腦上所保存的錢包。 <br /><br />在任何時候，我們建議您的線上錢包中只存放少許的狗狗幣。 <br />這裡有許多基於瀏覽器的錢包:",
+    "dgc-browser-modal-dogechain": "DogeChain Wallet", 
+    "dgc-browser-modal-holytransaction": "HolyTransaction", 
+    "dgc-browser-modal-blockio": "Block.io", 
+    "dgc-browser-modal-online-disclaimer": "<strong>注意：</strong> 狗狗幣不負責與維護這些線上錢包。 儘管我們相信背後的組織具有可信度，但請自行斟酌風險。", 
+    "dgc-ios-wallet-modal-important": "重要：", 
+    "dgc-ios-wallet-modal-view-only": "iOS 上的 MyDoge 僅能用來查看錢包。", 
+    "dgc-ios-wallet-modal-not-send": "MyDoge <strong>不能</strong> 發送狗狗幣. 所以您必须自行建立實際的狗狗幣錢包.", 
+    "dgc-ios-wallet-modal-understand": "我了解，開始吧！", 
+    "dgc-ios-wallet-modal-los-gehts": "點擊按钮前往 Appstore 頁面.", 
+    "dgc-learnmore-modal-title": "了解更多：", 
+    "dgc-learnmore-modal-dogeisfun": "狗狗幣是一種有趣且應用性快速成長的加密貨幣。", 
+    "dgc-learnmore-modal-crypto": "此種數位貨幣被稱為 \"加密貨幣\"。", 
+    "dgc-learnmore-modal-anonymous": "加密貨幣完全匿名、去中心化，且極其安全.", 
+    "dgc-learnmore-modal-wallet": "狗狗幣可以通過電腦、手機或網頁上的錢包來使用。",
+    "dgc-learnmore-modal-uses": "您可以使用它購買商品與服務也能用來交易成其他貨幣, 例如其他加密貨幣或是傳統法幣（美元, 新台幣等等）。", 
+    "dgc-learnmore-modal-tipping": "狗狗幣最常見的用途是 \"打賞\" 給創造或分享好内容的網友，您可以把它理解為更有意義的\"喜歡\" 或支持。打賞的狗狗幣可以全世界的網路上使用。", 
+    "dgc-learnmore-modal-very-easy": "狗狗幣的使用非常簡單。", 
+    "dgc-learnmore-modal-step1": "<strong>步驟一：</strong> 創建錢包。", 
+    "dgc-learnmore-modal-step2": "<strong>步驟二：</strong> 獲取狗狗幣。", 
+    "dgc-learnmore-modal-step3": "<strong>步驟三：</strong> 發送狗狗幣。", 
+    "dgc-learnmore-modal-step4": "<strong>步驟四：</strong> 關注最新狗狗幣動態。", 
+    "dgc-learnmore-modal-more-info": "下面的指南裡有更多的資訊。", 
+    "dgc-learnmore-modal-getting-started": "新手指南", 
+    "dgc-community-modal-title": "社群：", 
+    "dgc-community-modal-get-involved": "參與狗狗幣！", 
+    "dgc-community-modal-participate": "加入 Reddit 社群，在我們的 IRC 頻道聊天，或參與我們的論壇。", 
+    "dgc-community-modal-reddit-desc": " - 主要的狗狗幣社群。", 
+    "dgc-community-modal-twitter-desc": " - 官方狗狗幣 Twitter",
+    "dgc-community-modal-irc": "#dogecoin IRC", 
+    "dgc-community-modal-irc-desc": " - 基於狗狗幣的線上聊天室。", 
+    "dgc-community-modal-forum": "狗狗幣論壇",
+    "dgc-community-modal-forum-desc": " - 狗狗幣社群論壇。",
+    "dgc-community-modal-friendly": "我們樂於助人, 只要是狗狗幣相關的問題我們都樂於回答。", 
+    "dgc-community-modal-not-bite": "試看看嘛, 我們保證不咬您！", 
+    "dgc-exchanges-modal-title": "獲取狗狗幣：",
+    "dgc-exchanges-modal-many-ways": "有無數種方式獲取狗狗幣。 您可以透過購買、交易、接受獎勵，或是「挖礦」獲取狗狗幣！", 
```

Original sentence "There are numerous amounts of ways to get Dogecoin. You can buy them, trade for them, get tipped, \"mine\" them, and more. They are surprisingly endless!"
```suggestion
    "dgc-exchanges-modal-many-ways": "有無數種方式取得狗狗幣。 您可以透過購買、交易、接受獎勵、「挖礦」以及其他各種方式獲得狗狗幣。方法多到驚訝的永無止境！", 
```